python/tests/test_constraints.py

```diff
@@ -1139,9 +1139,9 @@ def test_constr_LBOUND(self):
 
             for t in range(self.T):
                 for bus in net.buses:
-                    self.assertEqual(u[bus.index_v_mag[t]],bus.v_max)
+                    self.assertEqual(u[bus.index_v_mag[t]],bus.v_max_reg)
                     self.assertEqual(u[bus.index_v_ang[t]],pf.BUS_INF_V_ANG)
-                    self.assertEqual(l[bus.index_v_mag[t]],bus.v_min)
+                    self.assertEqual(l[bus.index_v_mag[t]],bus.v_min_reg)
```

Comment:
Here you are using v_max/min_reg in the constraint LBOUND. I do not think this is correct. This constraint has nothing to do with regulation. I thought we agreed that only regulation constraints were going to use v reg limits.